apm-sniffer/apm-sdk-plugin/jdk-cross-thread-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdk/thread/CallableOrRunnableConstructInterceptor.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.apache.skywalking.apm.plugin.jdk.thread;
+
+import org.apache.skywalking.apm.agent.core.context.ContextCarrier;
+import org.apache.skywalking.apm.agent.core.context.ContextManager;
+import org.apache.skywalking.apm.agent.core.context.trace.AbstractSpan;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceConstructorInterceptor;
+
+/**
+ * @author carlvine500
+ */
+public class CallableOrRunnableConstructInterceptor implements InstanceConstructorInterceptor {
+    @Override public void onConstruct(EnhancedInstance objInst, Object[] allArguments) {
+        ContextCarrier contextCarrier = new ContextCarrier();
+        AbstractSpan span = ContextManager.createExitSpan("JDK/Thread/run", contextCarrier, Thread.currentThread().getName());
```

Comment:
The third parameter of the `createExitSpan` method is IP address,  and the `Thread.currentThread().getName()` isn't the IP address. I suggest that you should call `createLocalSpan`. 
Also, I suggest that the operation name of this span cloud be like `Thread/{CALLABLE_CLASS_NAME}`.

Reply:
I will try to change to 
`ContextManager.createLocalSpan("Thread/" + objInst.getClass().getName())`

Reply:
@ascrutae 
do you think it's better , this code is diffrent from CallableOrRunnableConstructInterceptor:
`ContextManager.createLocalSpan("Thread/" + objInst.getClass().getName() + "/" + method.getName())`